src/dataflow-types/src/client/controller.rs

```diff
@@ -122,11 +195,9 @@ where
                                     port_hint: 2102,
                                 },
                             ],
-                            // TODO: use `size` to set these.
-                            cpu_limit: None,
-                            memory_limit: None,
-                            // TODO: support sizes large enough to warrant multiple processes.
-                            processes: 1,
+                            cpu_limit: size_config.cpu_limit,
+                            memory_limit: size_config.memory_limit,
+                            processes: size_config.processes.get(),
```

Comment:
Can you update the `ServiceConfig` struct to take a `NonZeroUsize` directly?

Reply:
sure, will do